experiments/what-factors-caused-WDTK-to-grow.md

```diff
@@ -0,0 +1,45 @@
+# What factors caused WhatDoTheyKnow to grow?
+
+We’re often asked what caused WhatDoTheyKnow to grow so much and become the success it is today. As most of the original team who set up WhatDoTheyKnow have gone on to pastures new, the current Alaveteli team are often a bit stumped. 
+
+So, Gemma interviewed one of the WhatDoTheyKnow’s founders, Francis Irving, specifically about what they did to promote the site. 
+
+## Answer
+
+Summary of most important things:
+
+
+
+- Getting a **Google grant to do Google Adwords**. Advice: Make sure you appear on the search page of when people are searching for FOI especially. Before Google Adwords WDTK did not appear at all on search results, then after they got the grant they were listed above government adverts/sites. 
+
+
+- Tom Steinberg had lots of **contacts in the media** so got the site featured in mainstream media
```

Comment:
oh, and a full stop at the end 😃 (so hard to know whether to or not when dealing with bullet points!)